datalake/src/main/java/com/sequenceiq/datalake/entity/SdxDatabaseDrStatus.java

```diff
@@ -0,0 +1,102 @@
+package com.sequenceiq.datalake.entity;
+
+import java.util.UUID;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Table(name = "sdxDatabaseDrstatus", uniqueConstraints = @UniqueConstraint(columnNames = {"operationId"}))
```

Comment:
you shouldn't specify the name here

Reply:
This patch also has the sql file to create table "sdxDatabaseDrstatus". Without having the name here, how can I assume that the entity would using the table with the name "sdxDatabaseDrstatus"?

Reply:
will do.